MaxText/configs/base.yml

```diff
@@ -63,15 +63,20 @@ int8_training: False
 # base_mlp_dim, base_num_decoder_layers and/or head_dim.
 global_parameter_scale: 1
 base_emb_dim: 2560
+base_num_heads: 8
```

Comment:
Shouldn't be necessary? Use query and kv heads (and assert query_heads == kv_heads if appriopriate)

Reply:
Gotcha, gpt3 didn't differentiate query and kv heads and use the `num_heads` as well as `head_dims`. 

To avoid add new keys, will do something like 
```
assert num_query_heads == num_kv_heads
attention_layer = Gpt3MultiHeadAttention(
     num_heads=num_query_heads,
     ...
```

Reply:
Done.